src/Storage/Cookie.php

```diff
@@ -19,6 +19,21 @@ class Cookie implements StorageInterface {
      */
     private static $data = [];
 
+    /**
+     * When the cookie should expire (in seconds), defaults to 2 years
+     * @var integer
+     */
+    private $cookieExpiry = 63072000;
```

Comment:
Minor nit: Not sure if this is necessary if it is also set in the construct signature?